src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printIntro(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void main(String[] args) throws UnknownCommandException,EmptyInputException,java.io.IOException{
+        printIntro();
+        Scanner in = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        int count=0;
+        //Task[] list = new Task[100];
+        String line;
+        File f = new File("data/Duke.txt");
+        if(f.exists()){
+            Scanner s = new Scanner(f);
+            while(s.hasNext()){
+                String[] temp = s.nextLine().split("\\|");
+                if(temp[0].equals("[T] ")){
+                    list.add(new ToDo(temp[2]));
+                    count++;
+                    if(temp[1].equals(" [V] ")){
+                        list.get(count-1).markAsDone();
+                    }
+                } else if(temp[0].equals("[D] ")){
+                    list.add(new Deadline(temp[2], temp[3]));
+                    count++;
+                    if(temp[1].equals(" [V] ")) {
+                        list.get(count - 1).markAsDone();
+                    }
+                } else if(temp[0].equals("[E] ")){
+                    list.add(new Event(temp[2], temp[3]));
+                    count++;
+                    if(temp[1].equals(" [V] ")) {
+                        list.get(count - 1).markAsDone();
+                    }
+                }
+            }
+        } else {
+            f.createNewFile();
+        }
+        FileWriter fw = new FileWriter(f);
+        while(true) {
+            try {
+                line = in.nextLine();
+                if (line.equals("bye")) {
+                    break;
+                } else if (line.equals("list")) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Here are the tasks in your list:");
+                    for (int j = 0; j < count; j++) {
+                        System.out.println(j + 1 + "." + list.get(j).toString());
+                    }
+                    System.out.println("____________________________________________________________");
+                } else if (line.startsWith("done")) {
+                    String[] temp = line.split(" ");
```

Comment:
Consider using a more descriptive variable name.
```suggestion
                    String[] stringFragments = line.split(" ");
```